core/modules/taxonomy/taxonomy.api.php

```diff
@@ -272,6 +272,24 @@ function hook_taxonomy_term_view_alter(&$build) {
   $build['#post_render'][] = 'my_module_taxonomy_term_post_render';
 }
 
+/**
+ * Alter the results of taxonomy_term_page()
```

Comment:
Need a full-stop here (https://docs.backdropcms.org/doc-standards#general):

```suggestion
 * Alter the results of taxonomy_term_page().
```